new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Kubernetes.java

```diff
@@ -432,6 +434,48 @@ public static boolean deletePod(String name, String namespace) {
     return true;
   }
 
+  /**
+   * Returns the V1Pod object given the following parameters.
+   *
+   * @param namespace in which to check for the pod existence
+   * @param labelSelector in the format "weblogic.domainUID in (%s)"
+   * @param podName name of the pod to return
+   * @return V1Pod object if found otherwise null
+   * @throws ApiException if Kubernetes client API call fails
+   */
+  public static V1Pod getPod(String namespace, String labelSelector, String podName) throws ApiException {
+    V1PodList pods = listPods(namespace, labelSelector);
+    for (var pod : pods.getItems()) {
+      if (podName.equals(pod.getMetadata().getName())) {
+        return pod;
+      }
+    }
+    return null;
+  }
+
+  /**
+   * Get the creationTimestamp for a given pod with following parameters.
+   *
+   * @param namespace in which to check for the pod existence
+   * @param labelSelector in the format "weblogic.domainUID in (%s)"
+   * @param podName  name of the pod
+   * @return creationTimestamp from metadata section of the Pod
+   * @throws ApiException if Kubernetes client API call fail
+   */
+  public static String getPodCreationTimestamp(String namespace, String labelSelector, String podName)
+      throws ApiException {
+    DateTimeFormatter dtf = DateTimeFormat.forPattern("HHmmss");
+    // DateTimeFormatter dtf = DateTimeFormat.forPattern("YYYYMMDDHHmmss");
+    V1Pod pod = getPod(namespace, labelSelector, podName);
+    if (pod != null) {
+      // return pod.getMetadata().getCreationTimestamp().toString();
+      return dtf.print(pod.getMetadata().getCreationTimestamp());
+    } else {
+      logger.info("getPodCreationTimestamp(): Pod doesn't exist");
```

Comment:
we don't need the method name prefix since method name will be logged by default in the message

Reply:
I will fix it in my next PR.